common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -60,6 +61,33 @@ def test_add_entitlement_with_missing_data(self):
         )
         assert response.status_code == 400
 
+    def test_staff_user_required_for_post(self):
+        not_staff_user = UserFactory()
+        self.client.login(username=not_staff_user.username, password=TEST_PASSWORD)
+
+        course_uuid = uuid.uuid4()
+        entitlement_data = self._get_data_set(self.user, str(course_uuid))
```

Comment:
Nit/FYI: The actual data doesn't matter given that the serializers will never be executed.